src/api/app/models/user.rb

```diff
@@ -861,18 +861,11 @@ def to_param
   end
 
   def nr_of_requests_that_need_work
-    Rails.cache.fetch("requests_for_#{login}", expires_in: 2.minutes) do
-      nr_requests_that_need_work = 0
-
-      rel = BsRequestCollection.new(user: login, states: %w(declined), roles: %w(creator))
-      nr_requests_that_need_work += rel.ids.size
-
-      rel = BsRequestCollection.new(user: login, states: %w(new), roles: %w(maintainer))
-      nr_requests_that_need_work += rel.ids.size
-
-      rel = BsRequestCollection.new(user: login, roles: %w(reviewer), reviewstates: %w(new), states: %w(review))
-      nr_requests_that_need_work += rel.ids.size
-    end
+    #Rails.cache.fetch("requests_for_#{login}", expires_in: 2.minutes) do
```

Comment:
Could you add a comment why this got disabled? Or even enable it again.

Such things could also be mentioned in the commit message for
- the reviewer
- anyone (could even be you;-)) that later on stumbles over the code and wonders why a change was done